src/mumble/Log.ui

```diff
@@ -154,6 +154,26 @@
         </property>
        </widget>
       </item>
+      <item row="2" column="0">
+       <widget class="QCheckBox" name="qcbNoScope">
+        <property name="toolTip">
+         <string>If enabled TTS messages will not say (scope) before message</string>
+        </property>
+        <property name="text">
+         <string>Omit Message Scope</string>
+        </property>
+       </widget>
+      </item>
+      <item row="2" column="1">
+       <widget class="QCheckBox" name="qcbNoAuthor">
+        <property name="toolTip">
+         <string>If enabled TTS will not dictate who the message author</string>
```

Comment:
Should probably be `If enabled TTS will not dictate who the message author is` or `If enabled TTS will not dictate the message author`.

Reply:
With a comma even: `If enabled, TTS will not dictate...`.

Reply:
I agree.